### PR TITLE
changed caption again

### PR DESCRIPTION
**Pull request by andrew-scoppa**
Tried to change caption to something funny. Hope it works!

### CODE REVIEW

**Comment by MikeH-PRG** [comment]
looks good to me

**Comment by csportello** [comment]
I like it

File: _slides/02-andrew-scoppa.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-This is my favorite Inflatocat!
+This is my happy Inflato-Cat!
```
Comment (Gershatprg) [inline _slides/02-andrew-scoppa.md:7]: Mine too!

Comment (andrew-scoppa) [reply _slides/02-andrew-scoppa.md:7]: Thanks!

**Comment by mikemalinowski55** [comment]
Hello

**Pull request merged**